oauth2cli.go

```diff
@@ -30,6 +30,10 @@ type Config struct {
 	// If multiple ports are given, it will try the ports in order.
 	// If nil or an empty slice is given, it will allocate a free port.
 	LocalServerPort []int
+	// When set, the server will serve TLS traffic using the specified
+	// certificates. It's recommended that the public key's SANs contain
+	// the loopback addresses - 'localhost', '127.0.0.1' and '::1'
+	TLSConfig *TLSConfig
```

Comment:
This is ambiguous because it may be a config of the local server or oauth client. Also it may mistake `TLSConfig` and `tls.Config`. Instead add the fields `LocalServerCertFile` and `LocalServerKeyFile` here.

Reply:
Okay - in this case, should those two fields (`LocalServerCertFile`  and `LocalServerKeyFile`) be string literal or string pointer?
The only benefit of being string pointer is that it's easier to distinguish a field not being set (being `nil`) and wrongly set - empty string `""`.

Reply:
It is not necessary to distinguish nil and empty string here and those fields can be string literal.